denizen_scripts/survival/repo-link/spawn_scripts/grim_effects.dsc

```diff
@@ -4,13 +4,13 @@ grim_area_effects:
     on player enters spawn_grim_cuboid:
       - cast blindness duration:40t no_ambient hide_particles
       - time player 14000
-      - execute as_server "stopsound <player.name> music" silent
+      - adjust <player> stop_sound
```

Comment:
This is currently a spigot bug that passed into Denizen again like it did previously, while `execute` is usually a bad usage for the Denizen equivalent command, this one doesn't work
```suggestion
      - execute as_server "stopsound <player.name> music" silent
```